lib/jose-backward-compatibility.js

```diff
@@ -0,0 +1,7 @@
+// this file exists for backward compatibility only
```

Comment:
I'm fine dropping this file. There aren't enough users of this library to require it.